Octokit/Http/HttpClientAdapter.cs

```diff
@@ -109,6 +103,8 @@ protected virtual HttpRequestMessage BuildRequestMessage(IRequest request)
             {
                 var fullUri = new Uri(request.BaseAddress, request.Endpoint);
                 requestMessage = new HttpRequestMessage(request.Method, fullUri);
+
+                requestMessage.Properties["AllowAutoRedirect"] = request.AllowAutoRedirect;
```

Comment:
It felt like a nice feature to have, but I'm struggling to think of a scenario where I would actually need it.